src/poll/poll.dtos.ts

```diff
@@ -53,6 +59,14 @@ export abstract class ClientConfigBase {
 
 export class ClientConfigDiscordDto extends ClientConfigBase {
 
+    @IsNotEmpty()
+    @IsNumberString()
+    @ApiProperty({
+        description: 'Channel to post in',
```

Comment:
Server to post in?

Reply:
^ Small rename

Reply:
I thought I changed it, will double check